packages/@lwc/engine-dom/src/index.ts

```diff
@@ -40,6 +40,7 @@ export {
 
 // Engine-core public types ------------------------------------------------------------------------
 export type {
+    DataCallback,
```

Comment:
For consistency should we rebrand as `WireDataCallback`?
```suggestion
    DataCallback as WireDataCallback,
```

Reply:
Updated! - e50c18fa672334d7cabc8434217a8e059c242b6d